scipy/signal/spectral.py

```diff
@@ -400,9 +416,9 @@ def csd(x, y, fs=1.0, window='hann', nperseg=256, noverlap=None, nfft=None,
     return freqs, Pxy
 
 
-def spectrogram(x, fs=1.0, window=('tukey',.25), nperseg=256, noverlap=None,
+def spectrogram(x, fs=1.0, window=('tukey', .25), nperseg=256, noverlap=None,
```

Comment:
Would be good to add to the `Notes` a mention of the rationale behind the default window choice